pallets/gear/src/lib.rs

```diff
@@ -101,6 +102,14 @@ impl DebugInfo for () {
     }
 }
 
+#[derive(Clone, Debug, Decode, Encode, PartialEq, Eq, scale_info::TypeInfo)]
+#[cfg_attr(feature = "std", derive(serde::Deserialize, serde::Serialize))]
+pub struct GasInfo {
+    pub spent: u64,
+    pub to_send: u64,
+    pub burnt: u64,
```

Comment:
Let's think about names.
- `spent` is the gas limit enough to process the message
- `to_send` is the gas amount to be sent to other actors
- `burnt` is the gas amount that will be deducted from the sender's account for the message processing

So I'd rename `spent` to `min_limit` or similar, what do you think?

Reply:
Yes, it makes sense